src/core/issues-notes.js

```diff
@@ -23,111 +23,123 @@ export function run(conf, doc, cb) {
         "<div><h2>" + conf.l10n.issue_summary + "</h2><ul></ul></div>"
       ),
       $issueList = $issueSummary.find("ul");
-    $ins.each(function(i, inno) {
-      var $inno = $(inno),
-        isIssue = $inno.hasClass("issue"),
-        isWarning = $inno.hasClass("warning"),
-        isEdNote = $inno.hasClass("ednote"),
-        isFeatureAtRisk = $inno.hasClass("atrisk"),
-        isInline = $inno[0].localName === "span",
-        dataNum = $inno.attr("data-number"),
-        report = {
-          inline: isInline,
-          content: $inno.html(),
-        };
-      report.type = isIssue
-        ? "issue"
-        : isWarning ? "warning" : isEdNote ? "ednote" : "note";
-      if (isIssue && !isInline && !hasDataNum) {
-        issueNum++;
-        report.number = issueNum;
-      } else if (dataNum) {
-        report.number = dataNum;
-      }
-      // wrap
-      if (!isInline) {
-        var $div = $(
-          "<div class='" +
+    if (conf.githubAPI) {
+      Array.from($ins)
+        .filter(({ dataset: { number } }) => number && !ghIssues[number])
+        .forEach(issue => {
+          const { dataset: { number } } = issue;
+          const msg = `Github issue ${number} was closed on GitHub, so removing from spec`;
+          pub("warn", msg);
+          issue.remove();
+        });
+    }
+    $ins
+      .filter((i, issue) => issue.parentNode)
+      .each(function(i, issue) {
```

Comment:
Seems odd that it thinks you changed all these lines. Can you please ensure the formatting of all this doesn't change?

Reply:
Ignore this... I just saw your question below

Reply:
ok